apps/web/src/pages/brand/tabs/LayoutEditor.tsx

```diff
@@ -34,11 +34,16 @@ export function LayoutEditor({
     defaultValues: {
       content: layout?.content || '',
       name: layout?.name || '',
-      description: '',
+      description: layout?.description || '',
       isDefault: layout?.isDefault || false,
     },
   });
   const { mutateAsync: createNewLayout, isLoading: isLoadingCreate } = useMutation(createLayout);
+  const { mutateAsync: updateCurrLayout, isLoading: isLoadingUpdate } = useMutation<
```

Comment:
```suggestion
  const { mutateAsync: updateCurrentLayout, isLoading: isLoadingUpdate } = useMutation<
```
Probably a big nit on my side, but considering that `currying` is a JS thing, I don't like leaving only `Curr`.